src/content/developers/docs/transactions/index.md

```diff
@@ -305,6 +305,22 @@ Watch Austin walk you through transactions, gas, and mining.
 
 <iframe width="100%" height="315" src="https://www.youtube.com/embed/er-0ihqFQB0" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+## Typed Transaction Envelope
+
+Ethereum has different transaction types that specify what a transaction should do. For example, sending Ether to an address, deploying a contract, etc.  
+Ethereum now has a new transaction standard that was defined and created by Micah Zoltu in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718): Typed Transaction Envelope, which forms the basis for some new and other yet-to-be-developed features on Ethereum.
+
+Previously, Ethereum had one format for transactions. Each transaction consists of a nonce, gas price, gas limit, to address, value, data, v, r, and s. These fields are RLP-encoded, to look something like this:  
```

Comment:
As above, recommend removing the temporally sensitive wording and focusing on the current state of things, which is that we have 4 transaction types now:
Cross Chain Legacy
Chain Locked Legacy [EIP-155](https://eips.ethereum.org/EIPS/eip-155)
Chain Locked with Access Lists [EIP-2930](https://eips.ethereum.org/EIPS/eip-2930)
Chain Locked with Split Fees [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)